main/src/main/scala/sbt/nio/Watch.scala

```diff
@@ -493,6 +493,7 @@ object Watch {
   final val defaultInputOptions: Seq[Watch.InputOption] = Seq(
     Watch.InputOption("<enter>", "interrupt (exits sbt in batch mode)", CancelWatch, '\n', '\r'),
     Watch.InputOption(4.toChar, "<ctrl-d>", "interrupt (exits sbt in batch mode)", CancelWatch),
+    Watch.InputOption('l', "reload the build", Reload),
     Watch.InputOption('r', "re-run the command", Trigger),
```

Comment:
It might be slightly confusing since we have similar set of alphabet soup on load failure:

```
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
```

It might be more consistent if `r` reloaded the build, and `t` re-triggered the command?